www/src/pages/ar/why.md

```diff
@@ -0,0 +1,52 @@
+
+إذا كنت مهتمًا باستخدام Next.js بطريقة آمنة ، فهذا هو المكان المناسب للبدء. إذا كنت مهتمًا بأي من الخيارات التقنية المحددة التي اتخذناها ، فاقرأ :)
+
+## لماذا TypeScript ؟
+
+جافا سكريبت صعبة. لماذا نضيف المزيد من القواعد؟  
+لأننا نعتقد اعتقادًا راسخًا أن التجربة التي توفرها TypeScript ستدفعك إلى أن تكون مبرمجا أفضل، فتوفر لك الملاحظات مباشرة أثناء كتابة الكود من خلال تحديد أنواع البيانات المتوقعة(Types) ، هذا إلى جانب ميزة الإكمال التلقائي، و ينبهك في حال ارتكبت خطأَ كأن تحاول مثلا الوصول إلى خاصية غير موجودة أو تمرير قيمة من النوع الخطأ فيجبرك أن تصححها . سواء كنت مبتدئًا في تطوير الويب أو محترفًا متمرسًا ، فإن "صرامة" TypeScript ستوفر تجربة أقل إحباطًا وأكثر اتساقًا من Vanilla JS.
+
+هل لازلت غير مقتنع بـ Typesafety ؟ اذا شاهد هذا [أنت تستخدم Typescript بشكل خاطئ](https://www.youtube.com/watch?v=RmGHnYUqQ4k)
+
+## لماذا Next.js ؟
+
+نحن نحب React، بفضلها أصبح تطوير واجهات المستخدم UI سهل المنال بطريقة لم نكن نتخيلها. لقد أخذت بيد المطور وعبرت به أودية سحيقة صعبة.
+
+علي الرغم أن Next.js تقنية عنيدة قليلاَ، إلا أنها تقدم تجربة أقرب إلى الكمال في شتى النواحي routing و API حتي عرض الصور، فنحن نثق تمام الثقة أن Next.js ستقود المصورين الي الامام بقراراتها الحكيمة.
+
+## لماذا tRPC/Prisma/Tailwind وغيرها ؟
+
+بينما نؤمن بالحفاظ على بساطة الأشياء قدر الإمكان، فإن هناك قطع برمجية صغيرة تتكرر في كل تطبيق، `create-t3-app` يسمح لك بتبني هذه القطع بصورة رائعة.
+
+### لماذا tRPC ؟
+
+توفر tRPC تجربة سلسة في تطوير تطبيقات المستخدم (Client Side) كما في GraphQl، فإنها تتحايل على TypeScript لتوفر تجربة ممتازة للمطور.
+
+### لماذا Prisma ؟
+
+إن Prisma بالنسبة لـ SQL كـ Typescript بالنسبة لـ Javascript، فإنها تخلق تجربة مستخدم لم تكن موجودة من قبل، فتقوم بتوليد الانواع (Types) من Schema يقوم المطور بوضعها، ذلك إلى جانب توافقها مع [العديد من قواعد البيانات](https://www.prisma.io/docs/concepts/database-connectors)
+فبذلك تضمن Prisma الـ TypeSafety بين التطبيق وقاعدة البيانات
+
+توفر Prisma [مجموعة كاملة من الأدوات](https://www.prisma.io/docs/concepts/overview/should-you-use-prisma#-you-want-a-tool-that-holistically-covers-your-database-workflows) التي تجعل التفاعلات اليومية مع قاعدة البيانات أسهل.
+
+والجدير بالذكر أن Prisma Client مسؤول عن الاستعلام عن SQL وجعله أمرًا سهلاً للغاية لدرجة أنك بالكاد ستلاحظ أنك تستخدمه .
+
+ويكون Prisma Studio واجهة مستخدم رسومية ملائمة لقاعدة بياناتك والتي تتيح لك قراءة بياناتك ومعالجتها بسرعة دون الحاجة إلى كتابة التعليمات البرمجية.
+
+### لماذا Tailwind CSS ؟
+
+توفر لك Tailwind العناصر الانشائيه اللازمة من الوان وتباعد وغيرها، فتُسهل انشاء تطبيق جيد المظهر؟ وعلى عكس المكتبات الاخرى فانها لا تعيقك عندما تريد نقل تطبيقك الي مستوي اعلي و انشاء شئ جميل وفريد من نوعه يحمل عنك عبء اختيار أسماء للـ Classes و تنظيم الملفات.
+
+### لماذا NextAuth.js ؟
+
+إذا كنت تريد نظام مصادقة في تطبيق NextJS الخاص بك ، فإن NextAuth.js يعد حلاً ممتازًا دون الحاجة إلى إنشاءه بنفسك. فيأتي مزودًا بقائمة واسعة من الموفرين (Providers) إضافة مصادقة OAuth بسرعة ويوفر محولات (Adapters) للعديد من قواعد البيانات و ORMs.
```

Comment:
I am wondering is it clearer to use `مزودين` for `providers`? Because I associate مزودين with services more than موفر, and I am wondering if this is a me thing :thinking:

```suggestion
إذا كنت تريد نظام توثيق صالحيه في تطبيقك NextJS، فإن NextAuth.js يعد حلاً ممتازًا دون الحاجة إلى إنشاءه بنفسك. فيأتي مزودًا بقائمة واسعة من المزودين (Providers) إضافة مصادقة OAuth بسرعة ويوفر محولات (Adapters) للعديد من قواعد البيانات و ORMs.